src/components/block/ProgramCard/ProgramCard.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import ProgramCard from "./ProgramCard";
+import img from "./sample.jpg";
+
+export default {
+  title: "Block/ProgramCard",
+  component: ProgramCard,
+};
+
+const Template = ({ image, alt, title, description }) => (
+  <ProgramCard
+    image={image}
+    alt={alt}
```

Comment:
After looking at this here, I'm thinking that we may want to rename `alt` to `imageAlt` to make it clearer that it's the alt text that goes with the image. Since this component currently has props that let you control three different subelements, it's probably a good idea to be clear about which subelement is being affected.